util/VarText.h

```diff
@@ -179,6 +179,10 @@ class FO_COMMON_API VarText {
     static constexpr std::string_view METER_TYPE_TAG = "metertype";
     //! Variable value is the name of a registed ValueRef.
     static constexpr std::string_view FOCS_VALUE_TAG = "value";
+    //! Variable value is Planet::ID(), gives environment of that planet (Good, Poor...)
+    static constexpr std::string_view ENVIRONMENT_TAG = "environment";
+    //! Variable value is a user_string (translation string from <lang>.txt)
+    static constexpr std::string_view USER_STRING_TAG = "user_string";
```

Comment:
seems unrelated...?

Reply:
That is needed to implement agrrr3's suggestion for the SitRep (see en.txt):
`%planet% has been terraformed to %user_string% (%environment%).`